test/ctia/task/migration/migrate_es_stores_test.clj

```diff
@@ -847,45 +863,49 @@
 (defn load-test-fn
   [app maximal?]
   ;; insert 20000 docs per entity-type
-  (doseq [bundle (repeatedly 20 #(fixt/bundle 1000 maximal?))]
-    (POST-bulk app bundle))
-  (doseq [batch-size [1000 3000 6000 10000]]
-    (let [{:keys [get-in-config]} (helpers/get-service-map app :ConfigService)
-          services (app->MigrationStoreServices app)
-          conn (es-conn get-in-config)
-          total-docs (* (count @example-types) 20000)
-          _ (println (format "===== migrating %s documents with batch size %s"
-                             total-docs
-                             batch-size))
-          migration-id (format "test-load-%s" batch-size)
-          prefix (format "test_load_%s" batch-size)
-          start (System/currentTimeMillis)
-          _ (sut/migrate-store-indexes {:migration-id migration-id
-                                        :prefix       prefix
-                                        :migrations   [:__test]
-                                        :store-keys   (into [] example-types)
-                                        :batch-size   batch-size
-                                        :buffer-size  3
-                                        :confirm?     true
-                                        :restart?     false}
-                                       services)
-          end (System/currentTimeMillis)
-          total (/ (- end start) 1000)
-          doc-per-sec (/ total-docs total)
-          migration-state (ductile.doc/get-doc conn
-                                               (migration-index get-in-config)
-                                               "migration"
-                                               migration-id
-                                               {})]
-      (println "total: " (float total))
-      (println "documents per seconds: " (float doc-per-sec))
-      (doseq [[_ state] (:stores migration-state)]
-        (is (= 20000
-               (get-in state [:source :total])
-               (get-in state [:target :migrated]))))
-      (es-index/delete! conn (format "v%s*" prefix))
-      (ductile.doc/delete-doc conn "migration" migration-id {:refresh "true"})))
-  (es-index/delete! (es-conn (helpers/current-get-in-config-fn app)) "ctia_*"))
+  (try
+    (doseq [bundle (repeatedly 20 #(fixt/bundle 1000 maximal?))]
+      (POST-bulk app bundle))
+    (doseq [batch-size [1000 3000 6000 10000]
+            :let [{:keys [get-in-config]} (helpers/get-service-map app :ConfigService)
+                  services (app->MigrationStoreServices app)
+                  conn (es-conn get-in-config)]
+            migration-id (format "test-load-%s" batch-size)
+            prefix (format "test_load_%s" batch-size)]
```

Comment:
it belongs to the `let` doesn't it?
```suggestion
                  conn (es-conn get-in-config)
                  migration-id (format "test-load-%s" batch-size)
                  prefix (format "test_load_%s" batch-size)]]
```

Reply:
Gosh, yes!

Reply:
I wonder why this didn't fail? There's a s/Str schema on migration-id `get-migration`. I'll investigate.